arangod/Cluster/MaintenanceFeature.cpp

```diff
@@ -215,7 +215,7 @@ void MaintenanceFeature::initializeMetrics() {
                                  "Time spend execution the action [ms]"),
         metricsFeature.histogram(
             {StaticStrings::MaintenanceActionQueueTimeMs, action_label},
-            log_scale_t<uint64_t>(2, 82, 86400000, 12),
+            log_scale_t<uint64_t>(2, 82, 3600000, 12),
```

Comment:
@maierlars could you please comment on this? i was assuming that you had made the choice deliberately? ibm clusters?

Reply:
All I can say is that when I looked at the metrics output, the lower bound for the smallest queue time bucket was around 42000 ms.
Which is a bit too high I think, as the queue time should be a few ms only. So in my case all actions were in that smallest bucket, and the metric was somewhat useless.

Reply:
Should actions be queued for more than an hour, at all?

Reply:
it's possible. 4 core machine. and 2 big `ensureIndex` I assume

Reply:
At that time we thought one day would be appropriate as upper bound. However, if an action is queued longer than an hour we already have a problem. So I think this is fine.